recipes/linux.rb

```diff
@@ -19,7 +19,7 @@
 #
 
 # Install Nexpose Pre-Reqs
-package 'screen'
+package 'screen' unless node['platform_family'] == 'rhel' # rhel requires CD/ISO to be mounted to install screen
```

Comment:
@asebastian-r7 @kgawt-r7 This seems awfully opinionated in that it assumes no one outside of Rapid7 would use this cookbook on RHEL to install Nexpose.

All this nonsense can go away if Nexpose ships with a static linked binary of screen/tmux or has a new console which doesn't depend on a persistent virtual console. If you look at the static binary route, you will want to make sure there isn't any license conflicts though.